src/analyses/variable-sensitivity/constant_array_abstract_object.h

```diff
@@ -50,7 +52,8 @@ class constant_array_abstract_objectt:public array_abstract_objectt
 protected:
   CLONE
 
-  virtual bool merge(abstract_object_pointert other) override;
+  virtual abstract_object_pointert merge(
```

Comment:
It's only possible to `override` `virtual` methods, therefore `virtual` is not required here.